frontend/packages/console-telemetry-plugin/src/telemetry.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-console */
```

Comment:
can you move and rename this file to src/listeners/segment.ts as this is a specific implementation? Wdyt?